src/EnergyPlus/Psychrometrics.hh

```diff
@@ -1247,9 +1243,7 @@ namespace Psychrometrics {
 
 	inline
 	Real64
-	CPHW(
-		Real64 const Temperature // unused1208
-	)
+	CPHW()
```

Comment:
@mbadams5 Same here.  The better change would be to improve this function to return a variable CP.  Once all the calls to this are changed to take out T, it's a real pain to put it back.  And if we're content with a fixed value for CP, then this shouldn't be a function anymore, but a static constant.